static/src/javascripts/projects/common/modules/experiments/tests/clever-friend-brexit.js

```diff
@@ -1,31 +1,48 @@
 define([
 
 ], function() {
+    function noop() {}
+
+    function companionInteractives() {
+        return document.querySelectorAll('figure[data-canonical-url^="https://interactive.guim.co.uk/2016/05/brexit-companion/"]');
+    }
+
     return function () {
         this.id = 'CleverFriendBrexit';
```

Comment:
I think you need to change the id, otherwise people who were already put into buckets will stay there so you will get 50% of users who didn't come to the site since you started the test (and the existing 90% from before)